next-frontend/src/components/live/LiveResultsTable.tsx

```diff
@@ -0,0 +1,223 @@
+import { useMemo } from "react";
+import _ from "lodash";
+import { components } from "@/types/openapi";
+import events from "@/lib/wca/data/events";
+import { Link, Table } from "@chakra-ui/react";
+import {
+  centisecondsToClockFormat,
+  formatAttemptResult,
+} from "@/lib/wca/wcif/attempts";
+
+const advancingColor = "0, 230, 118";
+
+const customOrderBy = (
```

Comment:
How does the backend do similar tasks? If it never does: How does the current WCA Live do similar tasks?